Mathlib/Topology/Clopen.lean

```diff
@@ -102,6 +102,10 @@ theorem isClopen_inter_of_disjoint_cover_clopen {s a b : Set X} (h : IsClopen s)
   rintro x ⟨hx₁, hx₂⟩
   exact ⟨hx₁, by simpa [not_mem_of_mem_compl hx₂] using cover hx₁⟩
 
+theorem isClopen_of_disjoint_cover_clopen {a b : Set X} (cover : univ ⊆ a ∪ b)
```

Comment:
The name seems a bit strange; did you perhaps mean "... of disjoint_cover_open"?

Reply:
Hmm, I copied the name of the previous theorem, which involves an additional clopen set `s` ... changed now.